modules/effects/migrations/13_0_0/index.ts

```diff
@@ -65,6 +64,9 @@ function replaceEffectDecorators(
       if (!decorator) {
         return [];
       }
+      if (effect.initializer.getText().includes('createEffect')) {
```

Comment:
I just copy-pasted the code, and added this extra check so we don't wrap the effect in a `createEffect` block when `createEffect` is already used but the `@Effect` decorator wasn't manually removed. 

See this test for the behavior, https://github.com/ngrx/platform/blob/effects-migration/modules/effects/migrations/13_0_0/index.spec.ts#L215